packages/seedless-onboarding-controller/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/seedless-onboarding-controller`
+
+Backup and restore SRP
```

Comment:
Nit: Any more context we can provide here? Or maybe we can have this match what's in `package.json` at least:

``` suggestion
Backup and rehydrate SRP(s) using social login and password